tests/border-option.js

```diff
@@ -112,8 +112,10 @@ test('border style (custom ascii style)', t => {
 			topRight: '2',
 			bottomLeft: '3',
 			bottomRight: '4',
-			horizontal: '-',
-			vertical: '|',
+			top: '-',
+			bottom: '_',
+			left: '|',
+			right: '!',
```

Comment:
Typo

Reply:
I'm not sure what you meant there, I swapped top/bottom with left/right in the next push.

Reply:
Did you mean to use an exclamation mark instead of a `|`?

Reply:
No I meant to use 4 different characters.